### ISSUE TITLE
Boto3

### ISSUE THREAD

**Issue by Suncatcher** [pedropregueiro/migrate-ec2-secgroups#3]
Does it work with Boto3? If not, how to adapt it to make it work?

**Comment by pedropregueiro** [comment]
hey @Suncatcher, have you tried running it with Boto3? I'm not using AWS at the moment so I can't really test it, but if you do and it works, you can push a PR to update it, given it still works for Boto2 as well

**Comment by Suncatcher** [comment]
@pedropregueiro, yes, I tried and yes, it doesn't work with Boto3. They have completely rewrote the API.

**Comment by pedropregueiro** [comment]
according to Boto's documentation:
> Current Boto users can begin using Boto 3 right away. The two modules can live side-by-side in the same project

I'll make a few changes in the code to make it a bit more dynamic and then we can maybe try using boto3 to see if it works. will you be willing to help me test this with boto3 @Suncatcher?

**Comment by Suncatcher** [comment]
Okay, let's do it. But I already wrote [my own script](https://github.com/Suncatcher/aws_sg_migrate) to do this.
